pkg/consts/liqonet.go

```diff
@@ -4,7 +4,7 @@ const (
 	// NetworkManagerIpamPort is the port used by IPAM gRPCs.
 	NetworkManagerIpamPort = 6000
 	// NetworkManagerServiceName is the service name for IPAM gRPCs.
-	NetworkManagerServiceName = "liqo-network-manager"
+	NetworkManagerServiceName = "liqo-network-manager.liqo"
```

Comment:
I feel you shouldn't hard-code the service name/namespace here, as they may change at installation time (and that would be a very subtle bug to troubleshoot).